From eb9654388c1a8a8906589144365de22af65aebda Mon Sep 17 00:00:00 2001
From: Lu Hui <luhux76@gmail.com>
Date: Sun, 27 Aug 2023 15:41:49 +0800
Subject: [PATCH 2/7] hw: block: m25p80: add w25q128

---
 hw/block/m25p80.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/block/m25p80.c b/hw/block/m25p80.c
index dc5ffbc..07a4449 100644
--- a/hw/block/m25p80.c
+++ b/hw/block/m25p80.c
@@ -347,6 +347,7 @@ static const FlashPartInfo known_devices[] = {
     { INFO("w25q64",      0xef4017,      0,  64 << 10, 128, ER_4K) },
     { INFO("w25q80",      0xef5014,      0,  64 << 10,  16, ER_4K) },
     { INFO("w25q80bl",    0xef4014,      0,  64 << 10,  16, ER_4K) },
+    { INFO("w25q128",     0xef4018,      0,  64 << 10, 256, ER_4K) },
     { INFO("w25q256",     0xef4019,      0,  64 << 10, 512, ER_4K),
       .sfdp_read = m25p80_sfdp_w25q256 },
     { INFO("w25q512jv",   0xef4020,      0,  64 << 10, 1024, ER_4K),
-- 
2.41.0

